cmake/SundialsBuildOptionsPre.cmake

```diff
@@ -292,3 +292,7 @@ sundials_option(SUNDIALS_TEST_DEVTESTS BOOL
 # Include unit tests in regression tests
 sundials_option(SUNDIALS_TEST_UNITTESTS BOOL
   "Include unit tests in make test" OFF ADVANCED)
+
+sundials_option(ENABLE_CUSTOM_MPI_RUN_COMMAND BOOL "Enable passing in a custom command prefix to run when running MPI jobs" OFF ADVANCED)
```

Comment:
I think we can remove this option (see comment on the prefix variable).

Reply:
Changed to only SUNDIALS_TEST_MPIRUN_COMMAND.